object/virtual_machine.go

```diff
@@ -799,3 +799,15 @@ func (v VirtualMachine) UpgradeVM(ctx context.Context, version string) (*Task, e
 
 	return NewTask(v.c, res.Returnval), nil
 }
+
+// UUID is a helper to get the UUID of the virtual machine object.
+func (v VirtualMachine) UUID(ctx context.Context) string {
```

Comment:
Just curious, should this function return an error instead of an empty string?

Reply:
Good point, if we don't return the error, let's document the behavior in the godoc section.

Reply:
Generically speaking, a VM object always has a UUID, as the main purpose of this helper is to have an easy way to fetch it I think having a single `string` return to handle is better than handling a `(string, error)`.

But I absolutely agree that we should document the behavior in godoc 😄